portalnet/src/overlay_service.rs

```diff
@@ -1282,6 +1282,22 @@ where
                 )
             })?;
 
+        // Attempt to get semaphore permit if fails we return an empty accept
+        let permit = match self
+            .utp_controller
+            .inbound_utp_transfer_semaphore
+            .clone()
+            .try_acquire_owned()
+        {
+            Ok(permit) => permit,
+            Err(_) => {
+                return Ok(Accept {
+                    connection_id: 0,
+                    content_keys: requested_keys,
```

Comment:
```suggestion
                    content_keys: vec![],
```

Without this change won't the remote peer still try to connect to connection id 0?

Reply:
> it is empty already.

I see, thanks 👍